blocks/rss-alexa-feature-block/features/rss/xml.js

```diff
@@ -0,0 +1,161 @@
+import PropTypes from 'fusion:prop-types'
+import Consumer from 'fusion:consumer'
+import get from 'lodash/get'
+import moment from 'moment'
+import getProperties from 'fusion:properties'
+import { resizerKey } from 'fusion:environment'
+import { BuildContent } from '@wpmedia/feeds-content-elements'
+import { generatePropsForFeed } from '@wpmedia/feeds-prop-types'
+import { buildResizerURL } from '@wpmedia/feeds-resizer'
+
+const jmespath = require('jmespath')
+
+const cheerio = require('cheerio')
+
+const getURLType = (url) => {
+  const imgRegex = '/(jpe?g|png|webp|PNG|JPG|WEBP)$/'
+  const audioRegex = '^(https?|ftp|file)://(www.)?(.*?).(mp3)$'
+  const videoRegex = '^(https?|ftp|file)://(www.)?(.*?).(mp4|ts)$'
+  const uri = url.split('.')
+  const type = uri[uri.length - 1]
```

Comment:
You need to handle parameters.  What if there isn't a file extension on the link?